javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/javaparsermodel/contexts/LambdaExprContext.java

```diff
@@ -255,6 +263,9 @@ protected final Optional<Value> solveWithAsValue(SymbolDeclarator symbolDeclarat
     private int pos(MethodCallExpr callExpr, Expression param) {
         int i = 0;
         for (Expression p : callExpr.getArguments()) {
+            while (p instanceof EnclosedExpr) {
+                p = ((EnclosedExpr) p).getInner();
```

Comment:
Here p is an expression so you can write n.isEnclosedExpr()

Reply:
Good idea. Will change this to
```
            while (p.isEnclosedExpr()) {
                p = p.asEnclosedExpr().getInner();
            }
```